src/components/events/TournamentBracket.tsx

```
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
```

ESLint

[error] Replace `'react'` with `"react"`

(prettier/prettier)